scipy/optimize/_lsq/least_squares.py

```diff
@@ -726,6 +726,9 @@ def least_squares(
     if method == 'lm' and loss != 'linear':
         raise ValueError("method='lm' supports only 'linear' loss function.")
 
+    if method == 'lm' and abs_step is None:
```

Comment:
not None (this is causing tests to fail).

Another nitpick: back ticks are used to reference parameters in error messages (not followed everywhere, but at least in this file). Note that this is how parameters should be referenced in docstrings.